### PR TITLE
Added arrow function in below files

### PR DESCRIPTION
**Pull request by saravyas**
This PR  includes `arrow` function change.

### CODE REVIEW

File: attempt.js [hunk 1]
```diff
@@ -19,9 +19,9 @@ import isError from './isError.js'
  *   elements = []
  * }
  */
-function attempt(func, ...args) {
+const attempt = (func, ...args) => {
   try {
-    return func.apply(undefined, args)
+    return func(args)
   } catch (e) {
```
Comment (jdalton) [inline attempt.js]: 👆 That should be `func(...args)`.

**Comment by jdalton** [comment]
Thanks @saravyas!

I'm actually fine with these being function declarations though the removal of `.apply()` is fine. I know it kind of changes the intent of the PR though.

File: attempt.js (hunk 1 above)
Comment (saravyas) [reply attempt.js]: I made it to `spread operator`

File: conformsTo.js [hunk 2]
```diff
@@ -23,8 +23,6 @@ import keys from './keys.js'
  * conformsTo(object, { 'b': function(n) { return n > 2 } })
  * // => false
  */
-function conformsTo(object, source) {
-  return source == null || baseConformsTo(object, source, keys(source))
-}
+const conformsTo = (object, source) =>  source == null || baseConformsTo(object, source, keys(source))
```
Comment (jdalton) [inline conformsTo.js]: This can remain a declaration ☝️

File: conforms.js [hunk 3]
```diff
@@ -26,8 +26,6 @@ const CLONE_DEEP_FLAG = 1
  * filter(objects, conforms({ 'b': function(n) { return n > 1 } }))
  * // => [{ 'a': 1, 'b': 2 }]
  */
-function conforms(source) {
-  return baseConforms(baseClone(source, CLONE_DEEP_FLAG))
-}
+const conforms = (source) =>  baseConforms(baseClone(source, CLONE_DEEP_FLAG))
```
Comment (jdalton) [inline conforms.js]: This can remain a declaration ☝️

File: cond.js [hunk 4]
```diff
@@ -27,7 +27,7 @@ import map from './map.js'
  * func({ 'a': '1', 'b': '2' })
  * // => 'no match'
  */
-function cond(pairs) {
+const cond = (pairs) => {
   const length = pairs == null ? 0 : pairs.length
 
   pairs = !length ? [] : map(pairs, (pair) => {
```
Comment (jdalton) [inline cond.js]: This can remain a declaration ☝️

File: compact.js [hunk 5]
```diff
@@ -11,7 +11,7 @@
  * compact([0, 1, false, 2, '', 3])
  * // => [1, 2, 3]
  */
-function compact(array) {
+const compact = (array) => {
   let resIndex = 0
   const result = []
```
Comment (jdalton) [inline compact.js]: This can remain a declaration ☝️

File: cloneWith.js [hunk 6]
```diff
@@ -32,7 +32,7 @@ const CLONE_SYMBOLS_FLAG = 4
  * console.log(el.childNodes.length)
  * // => 0
  */
-function cloneWith(value, customizer) {
+const cloneWith = (value, customizer) => {
   customizer = typeof customizer == 'function' ? customizer : undefined
   return baseClone(value, CLONE_SYMBOLS_FLAG, customizer)
```
Comment (jdalton) [inline cloneWith.js]: This can remain a declaration ☝️

File: before.js [hunk 7]
```diff
@@ -13,12 +13,12 @@
...
  */
-function before(n, func) {
+const before = (n, func) => {
   let result
   if (typeof func != 'function') {
     throw new TypeError('Expected a function')
   }
-  return function(...args) {
+  return (...args) => {
     if (--n > 0) {
```
Comment (jdalton) [inline before.js]: 👆 That shouldn't be an arrow function because we need to forward the `this` in `func.apply(this, args)` below.

Comment (saravyas) [reply before.js]: Done ...

File: conformsTo.js (hunk 2 above)
Comment (jdalton) [reply conformsTo.js]: It's just personal preference for the primary function of the module to be a declaration instead of a function expression.

Comment (saravyas) [reply conformsTo.js]: Done

Comment (jdalton) [reply conformsTo.js]: Thanks, could you do it for the others I didn't explicitly call out too.

Comment (saravyas) [reply conformsTo.js]: I did it for what are all the files you requested below.

Comment (saravyas) [reply conformsTo.js]: I understood your request properly,I did it properly can you please `pull` the code and check it

File: cloneDeepWith.js [hunk 8]
```diff
@@ -32,7 +32,7 @@ const CLONE_SYMBOLS_FLAG = 4
  * console.log(el.childNodes.length)
  * // => 20
  */
-function cloneDeepWith(value, customizer) {
+const cloneDeepWith = (value, customizer) => {
   customizer = typeof customizer == 'function' ? customizer : undefined
   return baseClone(value, CLONE_DEEP_FLAG | CLONE_SYMBOLS_FLAG, customizer)
```
Comment (jdalton) [inline cloneDeepWith.js]: 👆 This can be left as a declaration

File: cloneDeep.js [hunk 9]
```diff
@@ -20,8 +20,7 @@ const CLONE_SYMBOLS_FLAG = 4
  * console.log(deep[0] === objects[0])
  * // => false
  */
-function cloneDeep(value) {
-  return baseClone(value, CLONE_DEEP_FLAG | CLONE_SYMBOLS_FLAG)
-}
+const  cloneDeep = (value) =>  baseClone(value, CLONE_DEEP_FLAG | CLONE_SYMBOLS_FLAG)
+
```
Comment (jdalton) [inline cloneDeep.js]: 👆 This can be left as a declaration

File: clone.js [hunk 10]
```diff
@@ -27,8 +27,7 @@ const CLONE_SYMBOLS_FLAG = 4
  * console.log(shallow[0] === objects[0])
  * // => true
  */
-function clone(value) {
-  return baseClone(value, CLONE_SYMBOLS_FLAG)
-}
+const clone = (value) => baseClone(value, CLONE_SYMBOLS_FLAG)
+
```
Comment (jdalton) [inline clone.js]: 👆 This can be left as a declaration

File: clamp.js [hunk 11]
```diff
@@ -15,7 +15,7 @@
  * clamp(10, -5, 5)
  * // => 5
  */
-function clamp(number, lower, upper) {
+const clamp = (number, lower, upper) => {
   number = +number
   lower = +lower
   upper = +upper
```
Comment (jdalton) [inline clamp.js]: 👆 This can be left as a declaration

File: chunk.js [hunk 12]
```diff
@@ -18,7 +18,7 @@ import slice from './slice.js'
  * chunk(['a', 'b', 'c', 'd'], 3)
  * // => [['a', 'b', 'c'], ['d']]
  */
-function chunk(array, size) {
+const chunk = (array, size) => {
   size = Math.max(size, 0)
   const length = array == null ? 0 : array.length
   if (!length || size < 1) {
```
Comment (jdalton) [inline chunk.js]: 👆 This can be left as a declaration

File: castArray.js [hunk 13]
```diff
@@ -30,7 +30,7 @@
  * console.log(castArray(array) === array)
  * // => true
  */
-function castArray(...args) {
+const castArray = (...args) => {
   if (!args.length) {
     return []
   }
```
Comment (jdalton) [inline castArray.js]: 👆 This can be left as a declaration

File: before.js [hunk 14]
```diff
@@ -13,7 +13,7 @@
  * jQuery(element).on('click', before(5, addContactToList))
  * // => Allows adding up to 4 contacts to the list.
  */
-function before(n, func) {
+const before = (n, func) => {
   let result
   if (typeof func != 'function') {
```
Comment (jdalton) [inline before.js]: 👆 This can be left as a declaration

File: attempt.js [hunk 15]
```diff
@@ -19,9 +19,10 @@ import isError from './isError.js'
  *   elements = []
  * }
  */
-function attempt(func, ...args) {
+const attempt = (func, ...args) => {
   try {
```
Comment (jdalton) [inline attempt.js]: 👆 `const attempt` can be left as a declaration

**Comment by jdalton** [comment]
Thank you for your patience and follow-ups @saravyas!

**Pull request merged**